recipes/met/runner.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+source $(pwd)/bin/config.sh
+BASEDIR=$(dirname $0)
+NAME=$(basename $BASEDIR)
+VERSION=$DATE
+ACL=private
+
+(
+    
+    cd $BASEDIR
+    rm -rf input && mkdir -p input
+    rm -rf output && mkdir -p output
+
+    #rm -rf input/attendance_raw.csv
+
+    files=$(axway_ls -nrt Met_Museum/Attendance | grep .csv | awk '{print $NF}')
+    echo $files
+    i=0
+    output_file="input/attendance_raw.csv"
+    for filepath in $files; do
+        echo $filepath
+        filename=$(basename $filepath)
+        echo $filename
+        axway_cmd get $filepath input/$filename
+        if [[ $i -eq 0 ]] ; then
+            # copy csv headers from first file
+            head -1 input/$filename > $output_file
```

Comment:
if we know the field headers are static & stable, there's no need to copy the csv headers, in the sql you can just remove "header" in the copy